### DISCUSSION TITLE
Mobile Safari freezes

### DISCUSSION THREAD

**Discussion by danny1605** [RaBa64/OpenDTU#10] [General]
go to the site with a mobile browser, in my case safari, the site freezes in after 5 sek. it is not possible to go to any setting because the page freezes .

[Video ](https://cloud.mandeltech.de/index.php/s/cqTtzsBEyqxX3NQ)

**Comment by danny1605** [comment]
The update did not fix the problem.

**Comment by MartinNK** [comment]
Hi ...

same with firefox mobile on android ... sometimes it freezes.
Diagram also not side scrollable on display ... to low space in both display modes ... vertical or horizontal

I recognized that if I change display in firefox to desktop mode there it shows graph not possible on that browser or something same meaning.

I hope get next update solves problem ... new opendtu version ...

Thanks ... great job
Martin

**Reply by MartinNK** [reply]
Hi Ralf,

any news about new version with database and new graphs without google?

Will database be compatible with old database from google graph version?

Is it possible restore old database then?

I will be very exited to test new version soon.

Thanks for your great work
Martin

**Reply by RaBa64** [reply]
Can you please check with the new calendar view.

**Reply by MartinNK** [reply]
Seems it is ok.
